src/encoded/static/components/cart/remove_multiple.js

```diff
@@ -29,7 +29,11 @@ const CartRemoveElementsModalComponent = ({ elements, closeClickHandler, removeM
         <Modal labelId="label-remove-selected" descriptionId="description-remove-selected" focusId="close-remove-items">
             <ModalHeader labelId="label-remove-selected" title="Remove selected datasets from cart" closeModal={closeClickHandler} />
             <ModalBody>
-                <p id="description-remove-selected">Remove the {elements.length} currently selected datasets from the cart. This action is not reversible.</p>
+                <p id="description-remove-selected">
+                    Remove the {elements.length} currently selected datasets from the cart. Series
```

Comment:
What is elements.length === 0 or 1?

Reply:
It can’t be 0 or this wouldn’t get called. It can certainly be 1.